content/Assets/Styles/base/_textarea.scss

```diff
@@ -0,0 +1,17 @@
+/* ==========================================================================
+   #TEXTAREA
+   ========================================================================== */
+
+textarea {
+    display: block;
+    @include rem(min-height, 12rem);
+    width: 100%;
+
+    @include themify($themes) {
+        border: 1px solid themed('color.text');
```

Comment:
If we have these border styles for textareas I feel like having the same ones defined on input and select would be sensible - maybe as a Sass placeholder which they all extend for ease of consistency.

Reply:
The `form-control` class actually applied a border-bottom. Felt like the `textarea` should have a border round the whole lot hence the separate declaration.

I think the forms as whole could do with some love, feels a bit messy to me at the moment.